composedtaskrunner-task-app-dependencies/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>composedtaskrunner-task-app-dependencies</artifactId>
+    <version>1.1.1.BUILD-SNAPSHOT</version>
```

Comment:
Need to check with @sobychacko.  When I changed the snapshots for the project to 1.0.0, the generated app pom would refer to 1.0.0 for the task-app-starters-core-dependencies.   Eventhough the parent for the project dependencies was set to 1.1.1.